src/components/themed/TransactionListRow.tsx

```diff
@@ -303,9 +305,10 @@ const getStyles = cacheStyles((theme: Theme) => ({
     color: theme.secondaryText
   },
   unconfirmedText: {
-    flexShrink: 1,
-    fontSize: theme.rem(0.75),
     color: theme.warningText,
     marginRight: theme.rem(1)
+  },
+  failedText: {
+    color: theme.dangerText
```

Comment:
If the `marginRight: 1` isn't needed here, shouldn't we remove it from  `unconfirmedText` too?

Reply:
I thought the marginRight is needed for unconfirmed text because the text can be longer with a confirmation count. Maybe it makes sense to move this marginRight to the base style for this component.